### ISSUE TITLE
Build getting failed with Unable to load Maven meta-data from https://jcenter.bintray.com/

### ISSUE THREAD

**Issue by rkaartikeyan** [software-mansion/react-native-screens#1274]
## Description

Build getting failed with `react-native-screens` package

### Screenshots
```FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:lintVitalRelease'.
> Could not resolve all artifacts for configuration ':react-native-screens:releaseRuntimeClasspath'.
   > Could not resolve com.facebook.react:react-native:+.
     Required by:
         project :react-native-screens
      > Failed to list versions for com.facebook.react:react-native.
         > Unable to load Maven meta-data from https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml.
            > Could not get resource 'https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml'.
               > Could not GET 'https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml'. Received status code 502 from server: Bad Gateway

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4m 30s
Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.9/userguide/command_line_interface.html#sec:command_line_warnings
95 actionable tasks: 95 executed
```
## Steps To Reproduce

1.

## Reproduction
`./gradlew clean && ./gradlew assembleRelese`

## Platform
- [x] Android

## Workflow
- [x] Bare workflow

## Package versions

```
  "dependencies": {
    "@react-navigation/bottom-tabs": "^6.0.9",
    "@react-navigation/elements": "^1.2.1",
    "@react-navigation/native": "^6.0.6",
    "@react-navigation/native-stack": "^6.2.5",
    "@reduxjs/toolkit": "^1.7.1",
    "axios": "^0.24.0",
    "react": "17.0.2",
    "react-native": "0.66.3",
    "react-native-gesture-handler": "^2.1.1",
    "react-native-reanimated": "^2.3.1",
    "react-native-safe-area-context": "^3.3.2",
    "react-native-screens": "^3.10.1",
    "react-native-status-bar-height": "^2.6.0",
    "react-native-video": "^5.2.0",
    "react-redux": "^7.2.6",
    "redux": "^4.1.2",
    "redux-logger": "^3.0.6",
    "redux-saga": "^1.1.3",
    "styled-components": "^5.3.3"
  },
  ```

**Comment by alaintalk** [comment]
i have the same error

**Comment by larsmunkholm** [comment]
![image](https://user-images.githubusercontent.com/6906782/149663120-6f90e4b7-2a3e-4335-a479-10d0d3904c7a.png)
Please do something about this. Jcenter is shutting down completely in February.

**Comment by mifi** [comment]
I thought they are going to keep a read only repo indefintely?

**Comment by larsmunkholm** [comment]
@mifi Status might have changed to "indefintely", but lately it has been down _a lot_, so I suspect they're not keen on people using their service.

**Comment by mifi** [comment]
Yes. I also came here due to 502 errors happening right now. I wonder what we can do about this as a workaround. Is there any mirror to jcenter?

**Comment by larsmunkholm** [comment]
jcenter() should be replaced with mavenCentral(), as stated by Gradle here https://blog.gradle.org/jcenter-shutdown and here https://docs.gradle.org/current/userguide/declaring_repositories.html#sub:maven_central

I've tried it out on react-native-svg, which has the same issue, and it seems to do the trick.

**Comment by kacperkapusciak** [comment]
Thanks for submitting an issue!

Due to the second major `jcenter()` outage in a week we've decided to remove it from `react-native-screens`.

With this, we've released patch version`3.10.2` that removes `jcenter()` 👉  https://github.com/software-mansion/react-native-screens/releases/tag/3.10.2
